src/main/java/org/mariadb/jdbc/internal/util/Utils.java

```diff
@@ -444,13 +444,6 @@ public static String nativeSql(String sql, boolean noBackslashEscapes) throws SQ
     public static Protocol retrieveProxy(final UrlParser urlParser, final ReentrantLock lock) throws QueryException, SQLException {
         switch (urlParser.getHaMode()) {
             case AURORA:
```

Comment:
Initialy driver permit only master/slave configuration for aurora, but there was different requests to permit having only one server (for integration environnement for example). 
If there is only one instance in cluster, the master-slave implementation will be used, resulting in trying to connect to a slave that will never be there. 
This "mono-instance" case must be handled.